baseline.yaml

```diff
@@ -125,6 +125,38 @@ criteria:
     control_mappings: # TODO
     scorecard_probe:
       - blocksDeleteOnBranches
+  - id: OSPS-TEST
+    maturity_level: 1
+    category: Build & Release
+    criteria: |
+      The project MUST use at least one automated test suite that
+      is publicly released as OSS (this test suite may be maintained
+      as a separate OSS project). It MUST clearly show
+      or document how to run the test suite(s).
```

Comment:
```suggestion
      The project MUST use at least one automated test suite that
      clearly documents when and how it is run.
```

Reply:
Do we define what an automated test suite is? Also, is the expectation here that no one would game this?  For example what would the automated test suite for baseline be?